integration-tests/tests/network/runner.rs

```diff
@@ -172,7 +172,7 @@ impl StateMachine {
                           _ctx: &mut Context<WaitOrTimeoutActor>,
                           _runner| {
                         if can_write_log.swap(false, Ordering::Relaxed) == true {
-                            debug!(target: "network", "runner.rs: Action {} {:?}", num_prev_actions, action_clone);
+                            debug!(target: "network", message =  "runner.rs: Action {} {}", num_prev_actions, ?action_clone);
```

Comment:
Note: the `{}` formatting specifies in the message are no longer necessary in invocations like these. When using the macro invocation with the old style, `tracing` will actually format in the fields into the message producing a log event like this (JSONish format):

```json
{ "target": "network", "message": "runner.rs: Action 42 Action { ... }" }
```

whereas with this new approach these values instead become keys on their own, and so `{}` no longer does anything (and is a part of the message string), resulting in an event like this instead:

```json
{ "target": "network", "message": "runner.rs: Action {} {}", "num_prev_actions": 42, "action_clone": "Action { ... }" }
```

Reply:
@nagisa Thanks, fixed.